data-es2016plus.js

```diff
@@ -824,7 +824,7 @@ exports.tests = [
           firefox53: firefox.sharedmem,
           firefox55: true,
           chrome48: chrome.sharedmem,
-          chrome61: true,
```

Comment:
Usually 2 Chrome pre-releases are shown in the table; people run tests on Canary before it reaches beta.